strawberry_django/auth/mutations.py

```diff
@@ -4,13 +4,17 @@
 from typing import TYPE_CHECKING, Any, Type, cast
 
 import strawberry
+from asgiref.sync import async_to_sync
+from channels import auth as channels_auth
```

Comment:
Just a small thing here. Since `channels` is an optional dependency, trying to import it here directly will raise an `ImportError` for anyone not using it.

We should import this inside a `try: except ImportError` and only try to use it when available

Reply:
Indeed, an important thing. Thx for noticing.
Ruff did force me to use another approach, but I've just pushed this change